rmgpy/rmg/model.py

```diff
@@ -1465,7 +1465,8 @@ def add_seed_mechanism_to_core(self, seed_mechanism, react=False):
             if not isNew:
                 logging.info("This library reaction was not new: {0}".format(rxn))
             elif self.pressure_dependence and rxn.elementary_high_p and rxn.is_unimolecular() \
-                    and isinstance(rxn, LibraryReaction) and isinstance(rxn.kinetics, Arrhenius):
+                    and isinstance(rxn, LibraryReaction) and isinstance(rxn.kinetics, Arrhenius) and (self.pressure_dependence.maximum_atoms is None  \
```

Comment:
could you refit these additions to the page size (lines are too long)?
otherwise, it looks gret, thanks!

Reply:
fixed